support/client/lib/vwf/view/blockly.js

```diff
@@ -451,11 +450,13 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
     function cleanUpdateToolbox( xml ) {
         // negateOverlap is being use to negate some of the effect of the MARGIN variable
         // in Blockly.createDom_ when deleting blocks over the flyout
-        var negateOverlap = 35;
+        // SJF: Overlap not present in Blockly with categories enabled
+
+        // var negateOverlap = 35;
         Blockly.updateToolbox( xml );
```

Comment:
Unfortunately you need to pick one and stick with it for the whole "game" - once it loads it must stay in a certain configuration. Haven't explored what reloading might mean/entail but I suspect it isn't pretty.